data/curated/enmoma/intro.md

```diff
@@ -0,0 +1,7 @@
+# English Monarchs and Marriages 
+
+This week we are exploring [English Monarchs and Marriages](https://github.com/frankiethull/english_monarch_marriages)! 
+
+> this dataset focuses on the names, ages, and marriages of various 'kings' and 'consorts'. the data ranges all the way back to 850 where the details are a bit fuzzy, spanning all the way to current day. names contain special characters; ages & years can be a bit of a regex puzzle as well. additionally, the age of kings and consorts may show quite a bit of an age gap. 
+
+The data was scraped from [Ian Visits](https://www.ianvisits.co.uk/articles/a-list-of-monarchs-by-marriage-6857/).
```

Comment:
```suggestion
The data was scraped from [Ian Visits](https://www.ianvisits.co.uk/articles/a-list-of-monarchs-by-marriage-6857/) by [f. hull](https://github.com/frankiethull), who also curated this week's post!
```

Reply:
See #725 for a ticket to make crediting more-automatic. Does this credit look ok, @frankiethull?